Example_04_05.py

```diff
@@ -0,0 +1,46 @@
+import sympy
+import numpy as np
+
+sympy.init_printing(use_latex='Mathjax')
```

Comment:
Since these examples are often run during the tests, it's probably better not to use pretty printing